README.md

```diff
@@ -26,30 +26,86 @@ cd ./yul-dc-management
 touch .secrets
 ```
 
-## If this is your first time working in this repo or the Dockerfile has been updated you will need to pull your services
+## Install
```

Comment:
The steps on line 23 (touch secrets) is no longer required (not in this diff, so I put the comment here